src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterValue } from '../../types/FilterValue';
+
+interface Props {
+  todos: Todo[],
+  setFilterValue: (value: string) => void,
+  filterValue: string,
+}
+
+export const Footer: FC<Props> = (props) => {
+  const {
+    todos,
+    setFilterValue,
+    filterValue,
+  } = props;
+
+  const remainingTodosLength = todos.filter(todo => !todo.completed).length;
+  const completedTodosLength = todos.filter(todo => todo.completed).length;
```

Comment:
Remaining todos should be taken from all the todos before filtering, not just currently rendered ones. Now while filtering by 'completed' the message shows 0

Reply:
Agreed, 0 items left looks weird 🤔

Reply:
done